README.md

```diff
@@ -0,0 +1,92 @@
+# mptcp-check
+A website to check the validity of a mptcp connection
+
+The website is build as a Flask app behind a lighttpd server.
+Everything is package in a docker container.
+
+## installing
+you will need to create the too following scripts. I would recommend the following. Do not forget to replace the `email` and `url`
+
+`start.sh`:
+``` bash
+#!/bin/bash
+email= #the email used by certbot
+url= #your domain name
+path=/var/docker/mptcp-check
+
+docker run --rm --name certbot-init \
+            -v "$path/cert:/etc/letsencrypt:rw" \
+            -p 80:80 \
+            certbot/certbot certonly \
+                --non-interactive --agree-tos \
+                --email $email \
+                -d $url \
+                --standalone
+
+docker run --name mptcp-check \
+            -v "$path/www:/var/www/:ro" \
+            -v "$path/cert/:/etc/letsencrypt/:ro" \
+            -p 80:80 -p 443:443 \
+            --restart unless-stopped \
+            --network host \
+            --detach \
+            mux514/mptcp-check:main
+```
+
+------------------------------------
+`renew.sh`
+``` bash
+#!/bin/bash
+email= #the email used by certbot
+url= #your domain name
+path=/var/docker/mptcp-check
+
+cert-before=$(sha256sum certbot/conf/live/$url/fullchain.pem)
```

Comment:
```suggestion
cert="certbot/conf/live/${url}/fullchain.pem"
cert_before=$(sha256sum "${cert}")
```

Don't use `-` in var names in bash, and use a var when a string is used twice to avoid typos